cvat-ui/src/components/quality-control/styles.scss

```diff
@@ -119,6 +103,8 @@ $excluded-background: #d9d9d973;
 .cvat-frame-allocation-table {
     @include cvat-table-dynamic-size;
 
+    height: calc(100vh - $grid-unit-size * 56);
```

Comment:
We shouldnt add a static height to table as we have `cvat-table-dynamic-size` now, it will break flex-grow property.
Instead, we should add `flex-grow:1` to `.cvat-table-wrapper` to ensure it takes all available place. And add a padding-bottom so pagination is not at the edge